taxa/taxonomy/taxonomydisplay.php

```diff
@@ -81,6 +81,7 @@ function displayTaxomonyMeta(){
 		<a href="taxonomydisplay.php"><b><?= $LANG['TAX_TREE_VIEWER'] ?></b></a>
 	</div>
 	<div id="innertext">
+		<h1 class="page-heading"><?php echo $LANG['TAX_DISPLAY'] . ': ' . $taxonDisplayObj->getTargetStr(); ?></h1>
```

Comment:
Change "Central Thesaurus" to being the h1 and change to "Central Taxonomic Thesaurus". "Taxonomy Display" not necessary
![image](https://github.com/BioKIC/Symbiota/assets/16688784/85ab292a-80a4-4669-82a1-2e9b6385ff5f)